uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/document-list.spec.js

```diff
@@ -180,7 +180,7 @@ define([
         });
 
         it('returns filtered document by expiry date of the document', function () {
-          expect(filteredDocumentList.length).toBe(2);
+          expect(filteredDocumentList.length).toBe(3);
```

Comment:
Why so? Mocks have been changed?

Reply:
No, the the previous test case was not correct.

Reply:
Did it pass before? Why it has been changed from 2 to 3?